cmake/modules/DetectMachine.cmake

```diff
@@ -20,22 +20,40 @@ ENDIF()
 MESSAGE("PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
 SET(LMMS_HOST_X86 FALSE)
 SET(LMMS_HOST_X86_64 FALSE)
+SET(LMMS_HOST_ARM32 FALSE)
+SET(LMMS_HOST_ARM64 FALSE)
+SET(LMMS_HOST_RISCV32 FALSE)
+SET(LMMS_HOST_RISCV64 FALSE)
 
 IF(NOT DEFINED WIN64 AND CMAKE_SIZEOF_VOID_P EQUAL 8)
-    SET(WIN64 ON)
+	# TODO: This seems a bit presumptous
+	SET(WIN64 ON)
```

Comment:
Do you want this to be defined on 64bit ARM?

Reply:
It's already defined on all 64-bit platforms that resolve to true in this statement.  This is just odd, presumptuous code, one in which my PR does not aim to fix.  The flag seems like it could be refactored out rather easily, hence the `TODO:`.